main.go

```diff
@@ -58,7 +61,34 @@ func failf(format string, v ...interface{}) {
 // Main
 //--------------------
 
-func main() {
+// Step ...
+type Step struct {
+	logger   logV2.Logger
+	xcpretty xcprettyinstaller.Installer
+}
+
+// NewStep ...
+func NewStep(logger logV2.Logger, xcpretty xcprettyinstaller.Installer) Step {
+	return Step{logger: logger, xcpretty: xcpretty}
+}
+
+func (s Step) ensureXCPretty() string {
```

Comment:
The name does not fully describe what the method does. I mean it can happen that after you call `ensureXCPretty` the the raw xcodebuild output will be used. Maybe `selectLogFormatter` or `determineLogFormatter` sounds better?

Reply:
Updated to `selectLogFormatter` in https://github.com/bitrise-steplib/steps-xcode-test-mac/pull/29/commits/f53eb3c74ad91035ffb8a092cbaaf58ebdab8d41.